nix/client.ovpn

```diff
@@ -0,0 +1,10 @@
+remote git.easycashmoney.org
+dev tun1
+port 1195
+comp-lzo
+tls-auth ta.key 1
+ca ca.crt
+key client-no-pass.key
```

Comment:
easyrsa build-client-full client-no-pass nopass
last part makes it so you dont have to enter a passphrase